.github/CODEOWNERS

```diff
@@ -22,21 +18,4 @@ pnpm-workspace.yaml @WordPress/openverse-frontend
 prettier.config.js  @WordPress/openverse-frontend
 .prettierignore     @WordPress/openverse-frontend
 
-automations/                @WordPress/openverse-maintainers
-brand/                      @WordPress/openverse-maintainers
-dead_links/                 @WordPress/openverse-maintainers
-docker/                     @WordPress/openverse-maintainers
-documentation/              @WordPress/openverse-maintainers
-readme_assets/              @WordPress/openverse-maintainers
-templates/                  @WordPress/openverse-maintainers
-utilities/                  @WordPress/openverse-maintainers
-.gitattributes              @WordPress/openverse-maintainers
-.gitignore                  @WordPress/openverse-maintainers
-.github/                    @WordPress/openverse-maintainers
-.devcontainer/              @WordPress/openverse-maintainers
-.pre-commit-config.yaml     @WordPress/openverse-maintainers
-CODE_OF_CONDUCT.md          @WordPress/openverse-maintainers
-CONTRIBUTING.md             @WordPress/openverse-maintainers
-justfile                    @WordPress/openverse-maintainers
-LICENSE                     @WordPress/openverse-maintainers
-README.md                   @WordPress/openverse-maintainers
+* @WordPress/openverse-maintainers
```

Comment:
We want to avoid using this `*` _all_ pattern because together with the previous lines it will make require reviews for more people than intended. Please leave the specific lines for the rest of files and folders associated with the `@WordPress/openverse-maintainers` group.

Reply:
If we put the all at the top, would the more specific patterns not over-ride them? Or does it just combine them?

If that is the case, then it would indeed be good to add the `notowned` check so we can check that all files are owned, but it adds a lot of complexity to this check because of the reasons I mentioned in my other comments.